### ISSUE TITLE
HTML "Contents" button should be styled link

### ISSUE THREAD

**Issue by rbeezer** [PreTeXtBook/pretext#471]
#424 makes an HTML `button` element into an active link using Javascript, but should really be a link, styled appropriately.  This would be desirable for accessibility.

**Comment by davidfarmer** [comment]
Under major revision.
